### PR TITLE
Add description for the new type of 'edit' fixers

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request

* Add a description for a new type of fixer that will edit a file.

#### Testing instructions

* Create a Jurassic Ninja site at https://jurassic.ninja/create/?shortlived
* Change your wp-config file to have the default placeholder strings as below:
```
define( 'AUTH_KEY',          'put your unique phrase here' );
define( 'SECURE_AUTH_KEY',   'put your unique phrase here' );
define( 'LOGGED_IN_KEY',     'put your unique phrase here' );
define( 'NONCE_KEY',         'put your unique phrase here' );
define( 'AUTH_SALT',         'put your unique phrase here' );
define( 'SECURE_AUTH_SALT',  'put your unique phrase here' );
define( 'LOGGED_IN_SALT',    'put your unique phrase here' );
define( 'NONCE_SALT',        'put your unique phrase here' );
define( 'WP_CACHE_KEY_SALT', 'put your unique phrase here' );
```
* Connect the site to Jetpack and purchase a plan that includes scan with credits
* Scan the site and verify that a threat was added because of the default salts.

## Testing wordpress.com

* Go to https://calypso.live/scan/{test-site}?branch=add/edit-fixer-support
* Click on the default salt threat to open it's details
* Verify that the text in the `How we will fix it?` section is `Jetpack Scan will edit the affected file or directory.`

## Testing cloud.jetpack.com 

* Checkout and build this branch locally
* Go to http://jetpack.cloud.localhost:3000/scan/{test-site}
* Verify as for wordpress.com above

## Screenshots

- Before:
Screen Shot 2020-11-03 at 14 41 28

- After:
Screen Shot 2020-11-03 at 15 11 18

### CODE REVIEW

File: client/components/jetpack/threat-item/utils.ts [hunk 1]
```diff
@@ -78,6 +78,8 @@ export const getThreatFix = ( fixable: ThreatFix ): i18nCalypso.TranslateResult
 				} );
 			}
 			return translate( 'Jetpack Scan will update to a newer version.' );
+		case 'edit':
+			return translate( 'Jetpack Scan will edit the affected file or directory.' );
```
Comment (a8ci18n) [inline client/components/jetpack/threat-item/utils.ts:82]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Jetpack Scan will replace the affected file or directory.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/edit-fixer-support)

**Review by monsieur-z — APPROVED** [review]
It all tests well 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5117626

Thank you @miguelxpn for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5117626).

**Pull request merged**